include/roboteam_world/net/netraw.h

```diff
@@ -1,49 +1,52 @@
 #ifndef _INCLUDED_NETRAW_H_
 #define _INCLUDED_NETRAW_H_
 
-#include <sys/types.h>
-#include <sys/socket.h>
-#include <netinet/in.h>
 #include <arpa/inet.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/types.h>
 
 #include <stdio.h>
 #include <string.h>
 
-namespace Net{
+namespace Net {
 
 //====================================================================//
 //  Net::Address: Network address class
 //  (C) James Bruce
 //====================================================================//
 
-class Address{
-  sockaddr addr;
-  socklen_t addr_len;
-public:
-  Address()
-    {memset(&addr,0,sizeof(addr)); addr_len=0;}
-  Address(const Address &src)
-    {copy(src);}
-  ~Address()
-    {reset();}
-
-  bool setHost(const char *hostname,int port);
-  void setAny(int port=0);
-
-  bool operator==(const Address &a) const
-    {return(addr_len==a.addr_len && memcmp(&addr,&a.addr,addr_len)==0);}
-  void copy(const Address &src)
-    {memcpy(&addr,&src.addr,src.addr_len); addr_len=src.addr_len;}
-  void reset()
-    {memset(&addr,0,sizeof(addr)); addr_len=0;}
-  void clear()
-    {reset();}
-
-  in_addr_t getInAddr() const;
-
-  void print(FILE *out = stdout) const;
-
-  friend class UDP;
+class Address {
+    sockaddr addr;
+    socklen_t addr_len;
+
+   public:
+    Address() {
+        memset(&addr, 0, sizeof(addr));
+        addr_len = 0;
+    }
+    Address(const Address &src) { copy(src); }
+    ~Address() { reset(); }
+
+    bool setHost(const char *hostname, int port);
+    void setAny(int port = 0);
+
+    bool operator==(const Address &a) const { return (addr_len == a.addr_len && memcmp(&addr, &a.addr, addr_len) == 0); }
+    void copy(const Address &src) {
+        memcpy(&addr, &src.addr, src.addr_len);
+        addr_len = src.addr_len;
+    }
+    void reset() {
+        memset(&addr, 0, sizeof(addr));
+        addr_len = 0;
+    }
```

Comment:
What... is this C?

Reply:
Basically. It's the library provided by SSL to listen to protobuf messages, so it's basically the best guarantee we have at receiving protobuf. It's the same as SSL-Vision uses.